src/citrine/resources/table_config.py

```diff
@@ -283,7 +265,7 @@ def get(self, uid: Union[UUID, str], version: Optional[int] = None):
             data['version'] = data['versions'][index]
         return self.build(data)
 
-    @deprecated(deprecated_in="0.124.0",
+    @deprecated(deprecated_in="0.124.0", removed_in="1.0.0",
```

Comment:
is adding `removed_in` different than just removing it?

Reply:
It is an indication of when we plan to remove it, and it shows up as part of the auto-generated docstring. But it's not the same as removal. This is just me being pedantic

Reply:
Why not remove it now? Since this is prepping for the 1.0.0 release

Reply:
We should leave time between marking as deprecated and removing. At least 2 weeks (https://citrine.atlassian.net/wiki/spaces/PLA/pages/440991870/Alpha%2BBeta%2BGA). That time might have passed, but 🤷